README.md

```diff
@@ -0,0 +1,20 @@
+# New Relic Helm charts repository
+
+This repository intends to host official Helm charts for New Relic's products or other charts developed by New Relic.
+
+## Developing a chart
```

Comment:
If we are not adding the `CHART_TESTING` content here, I think we should at least add a reference to it.

Reply:
I don't want to add a reference here to avoid cluttering the README with random information: it's suppose to be very very summarized. I'll mention it in the CONTRIBUTING file that is where the details about builds and other things are more extensive This document is linked in the section below.

Reply:
@alejandrodnm fixed in 8a5761